en/services/parameter.md

```diff
@@ -52,7 +57,13 @@ param.type = MAV_PARAM_TYPE_INT32;
 mavlink_msg_param_set_send(xxx, xxx, param.param_float, param.type, xxx);
 ```
 
-> **Note** A byte-wise conversion is needed, rather than a simple cast, to enable larger integers to be exchanged (e.g. 1E7 scaled integers can be useful for encoding some types of data, but lose precision if cast directly to floats).
+### Mavgen Python API (Pymavlink)
+
+Pymavlink does not include special support to byte-wise encode the non-float data types (unsurprisingly, because Python effectively "hides" low level data types from users).
+When working with a system that supports non-float parameters (e.g. PX4) you will need to do the encoding/decoding yourself when sending and receiving messages.
+
+There is a good example of how to do this in the Pymavlink [mavparm.py](https://github.com/ArduPilot/pymavlink/blob/master/mavparm.py) module (see `MAVParmDict.mavset()`).
```

Comment:
@tridge FYI, this is based on your recent commit https://github.com/ArduPilot/pymavlink/commit/f3cbc5391e984b1ba110ebe53f3de62b63cf6850

Reply:
@peterbarker, is this section OK?